import java.io.IOException;

public class Main {
    public static void main(String[] args) {
        ParallelArraySumary a = new ParallelArraySumary();
        a.Carregamento(9);
        a.Processamento(4);


//        Tester tester = new Tester();
//        try{
//            tester.executarTestes();
//        } catch (IOException e) {
//            throw new RuntimeException(e);
//        }
    }
}
